include/SFML/System/Angle.inl

```diff
@@ -257,3 +257,13 @@ constexpr Angle operator "" _rad(unsigned long long angle)
 }
 
 } // namespace Literals
+
+
+////////////////////////////////////////////////////////////
+// Static member data
+////////////////////////////////////////////////////////////
+
+// Note: the 'inline' keyword here is technically not required, but VS2019 fails
+// to compile with a bogus "multiple definition" error if not explicitly used.
+
```

Comment:
Should we remove this empty line, so it's clear the comments refer to the following definition?

Reply:
Sure.